### ISSUE TITLE
OpenMPI error

### ISSUE THREAD

**Issue by rajeev2010** [JuliaParallel/MPI.jl#4]
i have installed the package on ubuntu 12.04 LTS with MPI installed from synaptic package manager. the package compiles but all the tests fail. upon running first example individually i find the following error -

```
$ mpirun -np 2 ./juliampi ../examples/01-hello.jl

/home/rajeev/software/general/julia/julia: symbol lookup error: /usr/lib/openmpi/lib/openmpi/mca_paffinity_linux.so: undefined symbol: mca_base_param_reg_int
/home/rajeev/software/general/julia/julia: symbol lookup error: /usr/lib/openmpi/lib/openmpi/mca_paffinity_linux.so: undefined symbol: mca_base_param_reg_int
```

any ideas about the problem and fix.

**Comment by ViralBShah** [comment]
I suggest doing `Pkg.rm("MPI")`, making sure everything is removed, and try again using MPI.jl head.

**Comment by eschnett** [comment]
We recently solved an OpenMPI problem on Linux; see #70.

**Comment by eschnett** [comment]
I assume this issue has been solved.
